health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/BreadCrumbNew.js

```
@@ -0,0 +1,140 @@
+import React, { useEffect, useState, Fragment } from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+
+const BreadCrumbNew = (props) => {
+  const [expanded, setExpanded] = useState(false);
+  const [crumbsToDisplay, setCrumbsToDisplay] = useState([...props?.crumbs]);
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)